### ISSUE TITLE
[BUG] no module named promptflow.runtime

### ISSUE THREAD

**Issue by Kevinlee49** [microsoft/promptflow#2012]
**Describe the bug**
no module named promptflow.runtime

1.

>  2024-02-12 22:17:36 +0000  100090 promptflow_vectordb.tool INFO     [VectorIndexLookup] EmbeddingStore.Tool.Init started
> 2024-02-12 22:17:36 +0000  100090 promptflow_vectordb.tool INFO     [VectorIndexLookup] EmbeddingStore.Tool.LoadMLIndex started
> 2024-02-12 22:17:38 +0000  100090 promptflow_vectordb.tool INFO     [VectorIndexLookup] EmbeddingStore.Tool.LoadMLIndex completed
> Flow test failed with ResolveToolError: Tool load failed in 'search_question_from_indexed_docs': (ToolLoadError) Failed to load package tool 'Vector Index Lookup': (ModuleNotFoundError) No module named 'promptflow.runtime'

**Screenshots**
<img width="1107" alt="2024-02-12_16-17-233" src="https://github.com/microsoft/promptflow/assets/86863395/b4c21f19-fa8f-4bc5-8a7b-9d6eccef133c">

**Comment by 16oeahr** [comment]
This seems to be a duplicate of https://github.com/microsoft/promptflow/issues/788

**Comment by Adarsh-Ramanathan** [comment]
@Kevinlee49, the Vector Index Lookup tool has been deprecated, and is no longer under active development. Upgrading to the new Index Lookup tool should unblock you:

https://learn.microsoft.com/en-us/azure/ai-studio/how-to/prompt-flow-tools/index-lookup-tool

It should be a nearly direct replacement; with the one difference being that you'll need to pass in a string query instead of an embedding vector - the tool will internally embed the query if necessary.

**Comment by Kevinlee49** [comment]
> tool

Thank you @Adarsh-Ramanathan, what should I do with mlindex_content and query_type parts? do I need to make a json file like example for mlindex_content?
